src/pages/cli/migration/identity-claim-changes.mdx

```diff
@@ -0,0 +1,102 @@
+
+If you have an existing schema that uses owner-based `@auth` with an implicit identity claim rule on a type or field similar to this:
+
+```graphql
+type Todo @model @auth(rules: [{
+    allow: owner, # no explicit identity claim
+}]) {
+  id: ID!
+  title: String!
+}
+```
+
+Your GraphQL schema is using the default identity claim `"username"` and the Amplify CLI GraphQL Transformer is using the default value to generate your VTL files. Therefore, your schema is read as:
+
+```graphql
+type Todo @model @auth(rules: [{
+    allow: owner,
+    identityClaim: "username" # explicit identity claim
+}]) {
+  id: ID!
+  title: String!
+}
+```
+
+The Amplify CLI is changing the default to `"sub::username"` in the next major release, version 9.0.0, so if your `identityClaim` is not explicitly defined, the transformers will use `"sub::username"` unless you have set `"username"` to your schema as shown above.
+
+In the initial release, this functionality will be behind a feature flag, `useSubUsernameForDefaultIdentityClaim`, with `false` as the default value. Setting the feature flag to `true` enables the transformers to use the new identity claim; however, it is recommended to explicitly state your identity claim moving forward as this feature flag is only temporary (as shown above).
+
+### Manual changes to use `"sub::username"` for existing apps
+
+If you wish to migrate your VTL files before the changes in v9.0.0, set your `identityClaim` to `"sub::username"`.
+
+```graphql
+type Todo @model @auth(rules: [{
+    allow: owner,
+    identityClaim: "sub::username" # set your identity claim
+}]) {
+  id: ID!
+  title: String!
+}
+```
+
+Keep in mind that if you have existing owner records in your database and owner-reliant code in your code base, these changes will not migrate your data. The resolvers are backwards compatible, so your API will still be compatible with the former contract that uses `username` for the `owner` field. In other words, when using the `sub::usernam`e identity claim, your resolvers will authorize both `username` and `sub` values from the record that match the JWT, but the resolvers will write `<sub>::<username>` when no owner field input is specified.
+
+## What is the timeline?
+
+Version 8.1.0 of the Amplify CLI is introducing the feature flag, `useSubUsernameForDefaultIdentityClaim`, that will allow developers to opt into the `sub::username` default. New Amplify apps created will already be opted into the feature flag, but developers that want to use the new default will have to opt in. For existing Amplify apps, that do not have the feature flag, `useSubUsernameForDefaultIdentityClaim` will default to `false`.
+
+The Amplify CLI team will release version 9.0.0 in 30+ days, with the feature flag removed, and developers will need to manually set `"username"` to their schemas to maintain their former API contract, or the resolvers will start to store `sub::username` in their databases.
```

Comment:
The `30+` seems a bit arbitrary since a date isn't published here.

Reply:
Okay, I'll just mention it's the next major version